.eslintrc.yaml

```diff
@@ -1,10 +1,20 @@
+env:
+  browser: true
+  es2022: true
+  jest: true
+  node: true
```

Comment:
The items in this list tell eslint what global variables should be expected. (For example, `window` in the browser environment.) Node is included because some of the config files run under node and use node globals.